### PR TITLE
Export school analytics data to analytics database every hour

### PR DESCRIPTION
**Pull request by ethanmills**
## Ticket and context
I  have been dumping CSV files with this information far too regularly. It's time that stopped.

### Code quality checks
- [x] All commit messages are meaningful and true
- [ ] Added enough automated tests

### Gotchas
- [x] All `School` queries are correctly scoped - `eligible` when they need to be

## External API changes
None

### CODE REVIEW

File: app/services/analytics/ecf_school_service.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module Analytics
+  class ECFSchoolService
+    class << self
+      def update_school_analytics
+        return unless %w[test development production].include? Rails.env
```
Comment (jamesrose) [inline app/services/analytics/ecf_school_service.rb:7]: What's the motivation for this line?

Comment (kicferk1) [reply app/services/analytics/ecf_school_service.rb:7]: Saving money? 😂 

Do we have analytics database outside of those environments?

Comment (ethanmills) [reply app/services/analytics/ecf_school_service.rb:7]: yeah, those are the only environments the database exists

Comment (ethanmills) [reply app/services/analytics/ecf_school_service.rb:7]: And yes, the motivation for that is saving £100 per month per environment 😛

Comment (kicferk1) [reply app/services/analytics/ecf_school_service.rb:7]: 💰 💰 💰

**Review by kicferk1 — APPROVED** [review]

**Pull request merged**